src/main/java/seedu/address/model/tutorial/Tutorial.java

```diff
@@ -14,16 +14,18 @@ public class Tutorial {
     private final Group group;
     private final Content content;
     private final Time time;
+    private boolean status;
 
     /**
      * Constructs a {@code Tutorial}.
      * Every field must be present and not null.
      */
-    public Tutorial(Group group, Content content, Time time) {
+    public Tutorial(Group group, Content content, Time time, boolean status) {
         requireAllNonNull(group, content, time);
```

Comment:
Does `status` need to be nonNull here?

Reply:
I have excluded it because it is set to false by default, but for additional safety I will add it in. Thank you for pointing out!